.github/workflows/test_monitoring.yaml

```diff
@@ -0,0 +1,36 @@
+on:
+    push:
+      branches: [main]
+    pull_request:
+      branches: [main]
+
+name: Test monitoring
+
+permissions: read-all
+
+jobs:
+  test:
+      runs-on: ubuntu-latest
+      steps:
+          - name: Checkout repository
+            uses: actions/checkout@v4
+
+          - name: Install R
+            uses: r-lib/actions/setup-r@v2
+
+          - name: Set up environment
+            uses: ocha-dap/hdx-signals-actions@v2
+
+          - name: Test monitoring
+            shell: bash
+            run: |
+              shopt -s globstar
+              set -e
+              for x in test/automated/test_*.R; do Rscript "$x"; done
+            env:
+              HS_TEST: "TRUE"
```

Comment:
All good, just noting we'll want to change these once you merge the other branch to change these env names, so maybe merge that first then merge this branch!

Reply:
Yup those should be changed in the other branch!